app/main.py

```diff
@@ -1,3 +1,14 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    long_time_list = []
+
+    def inner(*args, **kwargs):
+
+        list_args = [*args]
+        if list_args in long_time_list:
+            print("Getting from cache")
+        else:
+            print("Calculating new result")
+        long_time_list.append([*args])
+        return func(*args, **kwargs)
+    return inner
```

Comment:
your approach is incorrect: you should store arguments and the result of the function with these arguments and return this result without calling function again when arguments are the same.
hint: use dictionary as storage